guide/index.adoc

```diff
@@ -0,0 +1,868 @@
+:pillow-docs-url: https://pillow.readthedocs.io/en/stable/
+:python-beginners-guide-url: https://wiki.python.org/moin/BeginnersGuide/Download
+:scale-url: https://ngageoint.github.io/scale/
+:seed-url: https://ngageoint.github.io/seed
+:seed-releases-url: https://github.com/ngageoint/seed-cli/releases
+:seed-version: 1.2.1
+:silo-url: https://github.com/ngageoint/seed-silo
+:pipenv-url: https://pipenv.pypa.io/en/latest/
+:venv-url: https://docs.python.org/3/library/venv.html
+
+== Purpose
+
+Scale and Seed optimize algorithm development from proof of concept to operational execution within distributed processing clusters.
+This guide introduces data scientists and algorithm developers to these technologies developed at the Research directorate of the National Geospatial-Intelligence Agency (NGA). 
+
+{scale-url}[Scale] is a distributed processing framework for near real-time processing of large data sets. 
+It frees data scientists and algorithm developers from the complexity of distributed system concepts and offers an integration platform for highly parallel processing of data. 
+Scale also ensures decision makers and system administrators are able to visually monitor data flows and ensure reliable data processing. 
+It enables a true DevOps environment for algorithm development and cloud transition.
+
+{seed-url}[Seed] is a general {seed-url}/seed.html[standard] to aid in the discovery and consumption of a discrete unit of work contained within a Docker image. 
+It is a JSON-defined manifest that is embedded into the Docker image as a label. 
+Seed allows data scientists to focus their expertise on solving domain problems and provides tools to ensure their algorithm is packaged properly for operational use. 
+Seed can be used throughout the development lifecycle to build, test and package within a simulated environment that closely emulates targeted operational environments. 
+This reduces risk and allows the algorithm requirements to be explicitly defined by the developers who best understand their processing needs. 
+Compliance with the Seed specification will be a requirement for algorithms to run in future versions of Scale.
+
+{silo-url}[SILO] (Seed Images Location Operation) is a REST API provided by the Seed team for discovering Seed images. 
+The API allows users to scan one or more repositories for Seed images and then search the resulting images and their manifests by keywords. 
+Frameworks such as Scale can use these endpoints to assist users in finding algorithm images and creating jobs out of them.
+
+The Seed command-line interface (CLI) is a standalone tool for working with Seed jobs locally. 
+This document will focus primarily on using the Seed CLI to build, test and publish your algorithm as a Seed-compliant Docker image. 
+The Seed toolchain consists of the CLI, SILO, {hashicorp-vault-url[Vault] (if using <<Secret Data>>) and a processing system that can consume Seed images, such as Scale. 
```

Comment:
In reality, Seed CLI doesn't do anything special regarding secrets. The `secret` `true`/`false` value is only there for when it moves into scale. Seed CLI treats all settings the same regardless of the value for `secret`. Maybe we just remove the reference to Vault here?

Reply:
How about adding a "full" qualifier; i.e. "The full Seed toolchain..." I think the idea here it to briefly touch on the end-to-end Seed ecosystem, even though, as mentioned in the preceding sentence, this guide focuses primarily on the CLI.

I'm fine with just removing the reference to Vault if you think that might be less confusing to new users..

Reply:
After thinking about this more, I removed the reference to Vault here. We do mention Vault later in the Secrets Data section.